internal/cmd/stack/stack_selector.go

```diff
@@ -47,6 +62,29 @@ func getStackID(cliCtx *cli.Context) (string, error) {
 	return got, nil
 }
 
+func stackExists(ctx context.Context, stackId string) error {
```

Comment:
Honestly I would change this to:
```suggestion
func stackExists(ctx context.Context, stackId string) (bool, error) {
```

Allowing the caller to choose what he wants to do with the information that the stacks exists or not. We could also get rid of the custom error then, as the caller could craft the error message itself.

Reply:
yup, agreed